docs/pages/careers.tsx

```diff
@@ -302,283 +321,252 @@ function renderFAQItem(index: number, defaultExpanded?: boolean) {
+      />
+      <AppHeaderBanner />
+      <AppHeader />
+      <main id="main-content">
+        <Section cozy bg="gradient">
+          <SectionHeadline
+            alwaysCenter
+            overline="Join us"
+            title={
+              <Typography variant="h2" component="h1">
+                Build <GradientText>the next generation</GradientText>
+                <br /> of tools for UI development
+              </Typography>
+            }
+            description="Together, we are enabling developers & designers to bring stunning UIs to life with unrivalled speed and ease."
+          />
+        </Section>
+        <Divider />
+        <OurValues />
+        <Divider />
+        {/* Perks & benefits */}
+        <Section bg="gradient">
+          <Grid container spacing={5} alignItems="center">
+            <Grid item md={6}>
+              <SectionHeadline
+                overline="Working at MUI"
+                title={
+                  <Typography variant="h2" id="perks-and-benefits">
+                    Perks & benefits
+                  </Typography>
+                }
+                description="To help you go above and beyond with us, we provide:"
+              />
+              <Box sx={{ maxWidth: 500 }}>
+                {[
+                  ['100% remote work', 'Our entire company is globally distributed.'],
+                  [
+                    'Retreats',
+                    'We meet up every 8 months for a week of working & having fun together!',
+                  ],
+                  [
+                    'Equipment',
+                    'We provide the hardware of your choice (initial grant of $2,500 USD).',
+                  ],
+                  ['Time off', 'We provide 33 days of paid time off globally.'],
+                ].map((textArray) => (
+                  <Box
+                    key={textArray[0]}
+                    sx={{ display: 'flex', alignItems: 'center', gap: 2, mt: 1, py: 0.5 }}
+                  >
+                    <IconImage name="pricing/yes" />
+                    <div>
+                      <Typography variant="body2" color="text.primary" fontWeight="semiBold">
+                        {textArray[0]}
+                      </Typography>
+                      <Typography variant="body2" color="text.secondary">
+                        {textArray[1]}
+                      </Typography>
+                    </div>
+                  </Box>
+                ))}
               </Box>
-            ))}
+            </Grid>
+            <Grid item xs={12} md={6}>
+              <Stack spacing={2} useFlexGap>
+                {companyInfo.map(({ title, description, routeUrl }) => (
+                  <Paper
+                    component={Link}
+                    href={routeUrl}
+                    noLinkStyle
+                    variant="outlined"
+                    sx={{ p: 2, width: '100%' }}
+                  >
```

Comment:
We were missing a `key` here, it generates warnings in the console:

<img src="https://github.com/mui/material-ui/assets/3165635/162ad622-52be-49b8-91de-3d73195bfb72" width="671">

Fixed in d4c5e8deac4626a0bc76cd4164bb633599ab79b1

Reply:
Sweet — thanks for the heads-up!